packages/graphback/src/resolvers/knex/resolverImplementation.ts

```diff
@@ -2,7 +2,7 @@ const knexContext = `context.db`
 const pubsub = `context.pubsub`
 
 export const createTemplate = (fieldName: string, tableName: string, typeName: string): string => {
-    return `${fieldName}: async (_, args, context) => {
+    return `${fieldName}: async (_: any, args: any, context: Context) => {
```

Comment:
Where is context going to be defined? Use something like `GraphBackContext | any`

Reply:
It will be defined in the context.ts in the template